src/js/components/SearchPage.js

```diff
@@ -140,23 +143,13 @@ export default function SearchPage() {
           </div>
         </div>
         <div className="row">
-          {isResourceSearch || !completedInitialLoad ? null : (
-            <div className="col-3 mt-3">
-              <FilterableFacet
-                results={facetOptions("topics")}
-                name="topics"
-                title="Topics"
-                currentlySelected={activeFacets["topics"] || []}
-                onUpdate={onUpdateFacets}
-              />
-              <FilterableFacet
-                results={facetOptions("department_name")}
-                name="department_name"
-                title="Department"
-                currentlySelected={activeFacets["department_name"] || []}
-                onUpdate={onUpdateFacets}
-              />
-            </div>
+          {isResourceSearch ? null : (
```

Comment:
this change to remove the check of `!completedInitialLoad` makes the filters flash for a second when doing a full-page refresh on the resource search (i.e. when looking at http://localhost:3000/search/?type=resourcefile)

Reply:
that may be fixed by https://github.com/mitodl/hugo-course-publisher/pull/254 but I think we should keep the `!completed InitialLoad` check for now anyhow

Reply:
Unfortunately the 'Apply Filters' popup closes as soon as a checkbox is clicked if `!completedInitialLoad` is kept here.  Since facets are going to be added to resources anyway I figured it would be okay to remove this.